common/lib/dependabot/pull_request_creator/message_builder.rb

```diff
@@ -142,6 +142,19 @@ def suffixed_pr_message_header
       end
 
       def message_trailers
+        return unless signoff_trailers || custom_trailer
+
+        [signoff_trailers, custom_trailer].compact.join("\n")
+      end
+
+      def custom_trailer
```

Comment:
Should we maybe. mention this is GitLab specific?

Reply:
In this case, having some custom git trailer is not necessarily a Gitlab specific logic. I think many workflows on any platform could do some actions based on presence of git trailer. It's just I happened to come by this usecase using Gitlab's changelog generation functionality 😄

Reply:
A bit more context on what other types of trailers someone might use from the git documentation itself: https://git-scm.com/docs/git-interpret-trailers

Reply:
Makes sense, cheers 👍